python-avd/pyavd/get_fabric_documentation.py

```diff
@@ -21,6 +23,8 @@ def get_fabric_documentation(
     topology_csv: bool = False,
     p2p_links_csv: bool = False,
     toc: bool = True,
+    digital_twin: bool = False,
+    digital_twin_global_config: dict | None = None,
```

Comment:
Instead of taking this as a dict here, would it not be better to handle the inheritance during generation of the metadata?